delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/error/PluginError.scala

```diff
@@ -0,0 +1,14 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.error
+
+trait PluginError extends SDKError
```

Comment:
make this a `sealed abstract class PluginError(reason: String, details: Option[String]) extends SDKError`
... and transform the `DependencyNotFound` into a case class with the appropriate information about what hasn't been found, like: `final case class DependencyNotFound(dependencies: Set[(Name, String)]) extends ...`